### PR TITLE
UIUX-74 New box deployment for caddy and frontend

### PR DESCRIPTION
**Pull request by EngineersBox**
## Changelog
* Added new ansible inventory, now two exist as `inventory/old.ini` and `inventory/prod.ini`
* Added new GH workflow input to specify which inventory to use (passed to `-i` with `ansible-playbook` command)
* Dockerized caddy deployment with gateway passthrough for API and frontend access
* Dockerized frontend via npm's `http-server` lib to serve on port 3002
* Refactored playbook to target new frontend build instead of old
* Refactored user pass provide to switch based on workflow target
* Refactored mentions of `elasticsearch to `opensearch`
* Added missing containers for each opensearch node
* Fixed dockerfile contexts
* Refactored backend dockerfile to have stricter copy-in for reduced size

### CODE REVIEW

**Review by XuguangSong98 — DISMISSED** [review]
looks good!

File: Dockerfile [hunk 1]
```diff
@@ -4,10 +4,19 @@ ENV PYTHONBUFFERED 1
...
+COPY summary summary
+COPY summary_1 summary_1
+COPY modules/RateLimiter modules/RateLimiter
 
 RUN python3 -m pip install --upgrade pip
 RUN python3 -m pip install --no-cache-dir -r requirements.txt
 RUN python3 -m spacy download en_core_web_trf
 
 EXPOSE 3001
+CMD ["gunicorn", "-w", "4", "-b", "0.0.0.0:3001", "app:app"]
```
Comment (ariadev79) [inline Dockerfile]: **[MINOR SUGGESTION]** End of line requested here and other files as well

**Review by ariadev79 — COMMENTED** [review]
All looks good to me. Just need end of line fr some of the files

**Comment by EngineersBox** [comment]
@WeiXinFam @alooba1 Have added the newlines, as per Fam's comment. Good for re-review

**Review by ariadev79 — APPROVED** [review]
All good to me

**Pull request merged**